CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## Your new git-extra command should support
+
+* OSX, Ubuntu, FreeBsd(You may need to browse their man page)
```

Comment:
Perhaps `OS X, Ubuntu, and FreeBSD (you may need to check the documentation for compatibility issues)`?